src/pages/EditMonitor.vue

```diff
@@ -1265,7 +1265,7 @@ message HealthCheckResponse {
                 this.monitor.body = JSON.stringify(JSON.parse(this.monitor.body), null, 4);
             }
 
-            if (this.monitor.type && this.monitor.type !== "http" && (this.monitor.type !== "keyword" || this.monitor.type !== "json-query")) {
+            if (this.monitor.type && (this.monitor.type !== "http" && this.monitor.type !== "keyword" && this.monitor.type !== "json-query")) {
```

Comment:
What you have changed is equivalent to this:

```suggestion
            if (this.monitor.type && this.monitor.type !== "http" && this.monitor.type !== "keyword" && this.monitor.type !== "json-query") {
```

Could you explain why (motivation) you did this change and what this fixes?

Reply:
There you go - it's like that now - I just figured that someone wanted the type checks in parentheses since they were used in the original code...